### PR TITLE
Add r-startr

### PR DESCRIPTION
**Pull request by jvegasbsc**

### CODE REVIEW

File: recipes/r-startr/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+source:
+  fn: {{ name }}-{{ version }}-{{ githash }}.tar.gz
+  url:
+    - https://earth.bsc.es/gitlab/es/{{name}}/repository/archive.tar.gz?ref=v{{ version }}
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script:
+    - set TARGET_ARCH=  # [win]
```
Comment (bgruening) [inline recipes/r-startr/meta.yaml]: Why is this needed?

**Comment by jdblischak** [comment]
@jvegasbsc Would you consider writing the recipe using the [conda_r_skeleton_helper](https://github.com/bgruening/conda_r_skeleton_helper)? It's easier for us to maintain the >800 R packages on conda-forge when they follow a similar pattern.

**Comment by jvegasbsc** [comment]
Yes, no problem. But it will be nice if you could add a reference to this conda_r_skeleton_helper in the README of this repository

**Comment by bgruening** [comment]
ping @jvegasbsc

**Comment by cbrueffer** [comment]
Added in #6432
